### PR TITLE
css-media-queries | Rocky Truong

### PR DESCRIPTION
**Pull request by rocky-truong**
image

image

image

### CODE REVIEW

File: css-media-queries/at-media.html [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  <style>
+
+    body {
+      min-height: 100vh;
+      background-color: palevioletred;
+    }
+
+    /* your code here 👇 */
+    @media (min-width: 768px) {
+      body { background-color: papayawhip; }
```
Comment (codingcodymiller) [inline css-media-queries/at-media.html:17]: Please always expand CSS rules to multiple lines

File: css-media-queries/breakpoints.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    .card-container {
+      padding: 0.5rem;
+      flex-basis: 100%;
+    }
+
+    /* your code here 👇 */
+    @media (min-width: 576px) and (max-width: 767px) {
+      .card-container { flex-basis: 50%; }
```
Comment (codingcodymiller) [inline css-media-queries/breakpoints.html:23]: Please always expand CSS rules to multiple lines

File: css-media-queries/breakpoints.html [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+    }
+
+    @media (min-width: 768px) {
+      .card-container { flex-basis: calc(100% / 3); }
```
Comment (codingcodymiller) [inline css-media-queries/breakpoints.html:27]: Please always expand CSS rules to multiple lines

File: css-media-queries/meta-viewport.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    }
+
+    .box {
+      width: 200px;
+      height: 200px;
+      background-color: blue;
+    }
+
+    @media (min-width: 568px) {
+      .box { background-color: red; }
```
Comment (codingcodymiller) [inline css-media-queries/meta-viewport.html:26]: Please always expand CSS rules to multiple lines

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by rocky-truong** [comment]
Made the changes requested so the CSS rules would have multiple lines.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**